### ISSUE TITLE
Unable to find yarp/dev/PreciselyTimed.h

### ISSUE THREAD

**Issue by GiulioRomualdi** [robotology/icub-main#601]
After this commit in YARP https://github.com/robotology/yarp/commit/e080845de607bbd264791c179a84babac9a650eb, it is not possible to compile `icub-main` in `devel`

This is the error
```
/home/gromualdi/robot-code/robotology-superbuild/robotology/ICUB/src/tools/controlBoardDumper/genericControlBoardDumper.h:23:10: fatal error: yarp/dev/PreciselyTimed.h: No such file or directory
 #include <yarp/dev/PreciselyTimed.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
The bug can be easily fixed with the following patch in [`genericControlBoardDumper.h`](https://github.com/robotology/icub-main/blob/devel/src/tools/controlBoardDumper/genericControlBoardDumper.h)
```patch
--- #include <yarp/dev/PreciselyTimed.h>
+++ #include <yarp/dev/IPreciselyTimed.h>
```

**Comment by GiulioRomualdi** [comment]
If you want I can open a PR with the fix

**Comment by pattacini** [comment]
Fixed in @d188e15
Thanks @GiulioRomualdi for reporting.
